security/openshift-security/confluent-platform-with-customSCC.yaml

```diff
@@ -0,0 +1,129 @@
+# In this example, 
+apiVersion: platform.confluent.io/v1beta1
+kind: Zookeeper
+metadata:
+  name: zookeeper
+  namespace: confluent
+spec:
+  replicas: 3
+  image:
+    application: confluentinc/cp-zookeeper:6.2.0
+    init: confluentinc/confluent-init-container:2.0.0
+  dataVolumeCapacity: 10Gi
+  logVolumeCapacity: 10Gi
+  podTemplate:
+    podSecurityContext:
+      fsGroup: 1001
+      runAsUser: 1001
+      runAsNonRoot: true
+---
+apiVersion: platform.confluent.io/v1beta1
+kind: Kafka
+metadata:
+  name: kafka
+  namespace: confluent
+spec:
+  replicas: 3
+  image:
+    application: confluentinc/cp-server:6.2.0
+    init: confluentinc/confluent-init-container:2.0.0
+  dataVolumeCapacity: 10Gi
+  metricReporter:
+    enabled: true
+  podTemplate:
+    podSecurityContext:
+      fsGroup: 1001
+      runAsUser: 1001
+      runAsNonRoot: true
+    resources:
+      requests:
+        cpu: "1"
+        memory: "4Gi"
+---
+apiVersion: platform.confluent.io/v1beta1
+kind: Connect
+metadata:
+  name: connect
+  namespace: confluent
+spec:
+  replicas: 2
+  image:
+    application: confluentinc/cp-server-connect:6.2.0
+    init: confluentinc/confluent-init-container:2.0.0
+  podTemplate:
+    podSecurityContext:
+      fsGroup: 1001
+      runAsUser: 1001
+      runAsNonRoot: true
+    resources:
+      requests:
+        cpu: "1"
+        memory: "4Gi"
+---
+apiVersion: platform.confluent.io/v1beta1
+kind: KsqlDB
+metadata:
+  name: ksqldb
+  namespace: confluent
+spec:
+  replicas: 2
+  image:
+    application: confluentinc/cp-ksqldb-server:6.2.0
+    init: confluentinc/confluent-init-container:2.0.0
+  dataVolumeCapacity: 10Gi
+  podTemplate:
+    podSecurityContext:
+      fsGroup: 1001
+      runAsUser: 1001
+      runAsNonRoot: true
+---
+apiVersion: platform.confluent.io/v1beta1
+kind: ControlCenter
+metadata:
+  name: controlcenter
+  namespace: confluent
+spec:
+  replicas: 1
+  image:
+    application: confluentinc/cp-enterprise-control-center-operator:6.1.1.0
+    init: confluentinc/cp-init-container-operator:6.1.1.0
```

Comment:
nit:: should it be consistent with other components?

Reply:
Fixed!